### PR TITLE
Added IDE section using codesandbox

### PR DESCRIPTION
**Pull request by 0-SiddharthJoshi-0**

### CODE REVIEW

**Review by arshadpatel — CHANGES_REQUESTED** [review]
I have reviewed your changes and I liked it but we are planning for simple page where the user can select the language and can execute the program easily... for example online GDB, programmiz and etc

File: frontend/IDE.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            <li><a href="#" class="btn">Sign Up</a></li>
+            <li><a href="#" class="btn">Login</a></li>
+        </ul>
+    </nav>
+    <section class="ide">
+        <h2>Online IDE</h2>
+        <p>Select a language to write and execute your code:</p>
+      
+        <p>Write, execute, and debug your code online:</p>
+        <iframe id="codeFrame" src="https://www.jdoodle.com/c-online-compiler" width="100%" height="500px" frameborder="0"></iframe>
```
Comment (0-SiddharthJoshi-0) [inline frontend/IDE.html:33]: I have changed the IDE now you can select which programming language you want to use, by using a searching in right next to execute button.

**Comment by 0-SiddharthJoshi-0** [comment]
I have changed the IDE as you requested .

**Review by arshadpatel — CHANGES_REQUESTED** [review]
Is there any other way instead of using sandbox... Because it is not looking good... As you can see there are two buttons of login one belongs to our website and other belongs to sandbox itself.

**Comment by 0-SiddharthJoshi-0** [comment]
I have tried using other IDE's but it makes page less responsive , and some of them show advertisements .

**Review by 0-SiddharthJoshi-0 — COMMENTED** [review]
.

**Pull request merged**